### PR TITLE
Add translation to Client-side_JavaScript_frameworks

### PR DESCRIPTION
**Pull request by bill0871**

### CODE REVIEW

**Comment by bill0871** [comment]
Add part of translation for #6269

File: files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.html [hunk 1]
```diff
@@ -133,13 +133,13 @@ <h2 id="冗长的DOM操作">冗长的 DOM 操作</h2>
   todoListEl.appendChild(frag);
 }</pre>
 
-<p>We've now got well over thirty lines of code dedicated <em>just</em> to the UI – <em>just</em> to the step of rendering something in the DOM – and at no point do we add classes that we could use later to style our list-items!</p>
+<p>我们现在已经有超过30行 <em>仅</em> 服务于 UI 的代码 – <em>只</em> 作用于在 DOM 中渲染某些内容的步骤 – 并且我们不会再添加类 - 随后可以用来设置列表项样式的类！</p>
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.html:136]:
```suggestion
<p>我们现在已经有超过30行<em>仅</em>服务于 UI 的代码——<em>只</em>作用于在 DOM 中渲染某些内容的步骤——并且我们不会再添加类，可以随后设置列表项样式的类！</p>
```

File: files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.html [hunk 2]
```diff
@@ -133,13 +133,13 @@ <h2 id="冗长的DOM操作">冗长的 DOM 操作</h2>
...
 
-<p>Working directly with the DOM, as in this example, requires understanding many things about how the DOM works: how to make elements; how to change their properties; how to put elements inside of each other; how to get them on the page. None of this code actually handles user interactions, or addresses adding or deleting a task. If we add those features, we have to remember to update our UI in the right time and in the right way.</p>
+<p>如本例所示，直接使用 DOM，我们需要了解有关 DOM 工作原理的许多内容：如何创建元素；如何改变它们的属性；如何将元素放在一起；如何让它们出现在页面上。这些代码实际上都没有处理用户交互，或添加/删除任务。如果我们添加这些功能，我们必须记得在正确的时间以正确的方式更新我们的 UI。</p>
 
-<p>JavaScript frameworks were created to make this kind of work a little easier — they exist to provide a better <em>developer experience</em>. They don't bring brand-new powers to JavaScript; they give you easier access to JavaScript's powers so you can build for today's web.</p>
+<p>JavaScript 框架的出现是为了使这些工作变得更容易 — 它们的存在是为了提供更好的 <em>开发人员体验</em>。它们没有给 JavaScript 带来新的功能；但它们使你可以更轻松地使用 JavaScript，因此你可以构建当下的web。</p>
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.html:140]:
```suggestion
<p>JavaScript 框架的出现是为了使这些工作变得更容易——它们的存在是为了提供更好的<em>开发体验</em>。它们没有给 JavaScript 带来新的功能；但它们使你可以更轻松地使用 JavaScript 来构建现代的 web。</p>
```

**Review by yin1999 — APPROVED** [review]
Thanks a lot. Welcome aboard

**Pull request merged**